### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by luis-pereira**

### CODE REVIEW

**Comment by agaida** [comment]
Apply, build and run great. LGTM -- we should mention somewhere in the wiki that the minimum version for LXQt is now 5.5

File: lxqtsettings.cpp [hunk 1]
```diff
@@ -80,7 +80,8 @@ class LXQt::SettingsPrivate
                     if (dir.cd(mParent->organizationName()) && dir.exists(file_name))
                     {
                         QSettings system_settings{dir.absoluteFilePath(file_name), QSettings::IniFormat};
-                        for (const QString & key : system_settings.allKeys())
+                        const QStringList keys = system_settings.allKeys();
+                        for (const QString & key : const_cast<const QStringList&>(keys))
```
Comment (palinek) [inline lxqtsettings.cpp]: The `const_cast` is unneeded here, `keys` variable is already `const`

Comment (luis-pereira) [inline lxqtsettings.cpp]: Two solutions for the same problem.

**Comment by agaida** [comment]
@luis-pereira @palinek - ready?

**Comment by palinek** [comment]
> - ready?
yes

**Pull request merged**